WootricSDK/WootricSDK/Wootric.m

```diff
@@ -298,4 +300,22 @@ + (void)setLogLevelVerbose {
   [WTRLogger setLogLevel:WTRLogLevelVerbose];
 }
 
+#pragma mark - Notifications
+
++ (NSNotificationName)surveyWillAppearNotification {
+  return @"com.wootric.surveyWillAppearNotification";
+}
+
++ (NSNotificationName)surveyWillDisappearNotification {
+  return @"com.wootric.surveyWillDisappearNotification";
+}
+
++ (NSNotificationName)surveyDidAppearNotification {
+  return @"com.wootric.surveyDidAppearNotification";
+}
+
++ (NSNotificationName)surveyDidDisappearNotification {
+  return @"com.wootric.surveyDidDisappearNotification";
```

Comment:
What is `com` short for?

Reply:
It's a convention in iOS: [Reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) :)

Reply:
Oh wow, I had no idea!

Reply:
TIL!